elfeed-org.el

```diff
@@ -172,13 +172,16 @@ all.  Which in my opinion makes the process more traceable."
   ;; Convert org structure to elfeed structure
   (-each (rmh-elfeed-org-import-headlines-from-files files tree-id)
     (lambda (headline)
-      (let ((text (car headline)))
+      (let* ((text (car headline))
+             (hyperlink (s-match "^\\[\\[\\(http.+?\\)\\]\\(?:\\[.+?\\]\\)?\\]" text)))
+        (when hyperlink
+          (rmh-elfeed-org-export-feed (push (nth 1 hyperlink) (cdr headline))))
```

Comment:
If I am not mistaken the ```push``` here will change the headline. It wouldn't break anything in this function now, but it might become a problem for later code. I fixed this in https://github.com/remyhonig/elfeed-org/pull/14 btw